demo/python/porcupine_demo.py

```diff
@@ -185,13 +192,13 @@ def _default_library_path():
     if args.show_audio_devices_info:
         PorcupineDemo.show_audio_devices_info()
     else:
-        if not args.keyword_file_path:
-            raise ValueError('keyword file path is missing')
+        if not args.keyword_file_paths:
+            raise ValueError('keyword file paths are missing')
 
         PorcupineDemo(
             library_path=args.library_path,
             model_file_path=args.model_file_path,
-            keyword_file_path=args.keyword_file_path,
+            keyword_file_paths=args.keyword_file_paths.split(','),
```

Comment:
do this so to also strip the whitespaces: `[x.strip() for x in my_string.split(',')]`

Reply:
good point. addressed.